urls_parsing.py

```diff
@@ -0,0 +1,65 @@
+"""Open source-file, parse Urls and write parsed Urls to timestamp directory"""
+
+import argparse
+import re
+import os
+
+from datetime import datetime
+
+
+def create_parse_args():
+    """Create command line arguments"""
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '-s', '--source_file',
+        help='path to file with urls you need to parse'
+    )
+    parser.add_argument(
+        '-n', '--number_of_urls',
+        default=2000,
+        help='portion of urls in the list to write to file'
+    )
+    args = parser.parse_args()
+    return args
+
+
+def create_timestamp_directory(directory_path):
+    """Create directory for output with timestamp"""
```

Comment:
Please add param description to the docstring i.e.:
```
:param directory_path: param desc
:type directory_path: str
`return value description and rtype 
```

Reply:
Please rewrite all function using following pattern:
def create_timestamp_directory(directory_path: str) -> str: